index.html

```diff
@@ -1,12 +1,12 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, shrink-to-fit=0, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
-  <title>glsl-sandbox</title>
-  <script src="https://lygia.xyz/resolve.js"></script>
-</head>
-<body style='margin:0'>
-  <script src="bundle.js"></script>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, shrink-to-fit=0, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <title>glsl-sandbox</title>
+    <script src="https://lygia.xyz/resolve.js"></script>
```

Comment:
If we are using vite maybe we could add this [lygiaResolver](https://www.npmjs.com/package/resolve-lygia) as devDependecy, right?

Reply:
Yes! We could also just straight out copy Lygia as a submodule and using [vite-glsl-plugin](https://www.npmjs.com/package/vite-plugin-glsl) include shader files. This is a more personal approach though, since I like looking at the files directly in my editor. `lygiaResolver` is of great  use here, I didn't know about it.